docs/guides/56-security/access-control.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Access Control
+---
+
+Databend incorporates both [Role-Based Access Control (RBAC)](https://en.wikipedia.org/wiki/Role-based_access_control) and [Discretionary Access Control (DAC)](https://en.wikipedia.org/wiki/Discretionary_access_control) models for its access control functionality. This guide describes the related concepts and provides instructions on how to manage access control in Databend.
+
+## Basic Concepts
+
+When a user accesses a data object in Databend, they must be granted appropriate privileges or roles, or they need to have ownership of the data object. A data object can refer to various elements, such as a database, table, view, stage, or UDF.
+
+![Alt text](../../public/img/guides/access-control-1.png)
```

Comment:
Can we unifom the picture style as:
![image](https://github.com/datafuselabs/databend-docs/assets/172204/cab6c82e-64a7-431c-917d-d7e3a1b7516c)

https://databend-git-access-control-guide-databend.vercel.app/guides/load-data/stage/stage-files

Reply:
The imported image address starts with @site to prevent static resource errors caused by changes in document locations  @soyeric128

Reply:
Pictures will be updated later.